test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -105,14 +82,40 @@ class MstPipelineTest : public AcceptanceFixture {
    */
   auto makeGetPendingTxsQuery(const std::string &creator,
                               const crypto::Keypair &key) {
-    return shared_model::proto::QueryBuilder()
-        .createdTime(iroha::time::now())
-        .creatorAccountId(creator)
-        .queryCounter(1)
-        .getPendingTransactions()
-        .build()
-        .signAndAddSignature(key)
-        .finish();
+    return complete(baseQry(creator).getPendingTransactions(), key);
+  }
+
+  /**
+   * Query validation lambda - check that empty transactions response returned
```

Comment:
It's not a lambda, but a full method. Maybe just "Check that empty..." as a first line would be better?

Reply:
It's meaning to be a query validation lambda. Nothing to fix.